uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/contact-actions-menu.js

```diff
@@ -0,0 +1,40 @@
+var Promise = require('es6-promise').Promise;
+var page = require('./page');
+
+module.exports = (function () {
+  var contactActionsMenuButton = '#crm-contact-actions-link';
+  var deleteUserAccountButton = '[data-delete-user-url]';
+
+  return page.extend({
+
+    /**
+     * Clicks the contact action menu button
+     */
+    openContactActionMenu: function () {
+      var casper = this.casper;
+
+      return new Promise(function (resolve) {
+        casper.then(function () {
+          casper.click(contactActionsMenuButton);
+          casper.wait(5000);
```

Comment:
Why do we use `wait`? (also applies to all places in this PR with `wait` usage)

Reply:
just waiting few seconds before taking a screenshot.  I have checked, it works without `wait()` as well. So, removed them.

Reply:
Ok, but **why** it was added beforehand? I am asking because there are major 2 issues with wait():
1) If a computer is fast enough, these 5 seconds will still add up and may form a couple of extra minutes in total.
2) The bigger problem if a computer is slow and wait() is actually needed, then there is a chance the scenario will break.

Reply:
> Ok, but why it was added beforehand?

I thought its good to wait for few seconds after an action is performed, before taking a screenshot. So, I added the `wait()`. Didn't realize its cons. Thanks @igorpavlov  for making it clear on its usage.